CONTRIBUTING.md

```diff
@@ -0,0 +1,141 @@
+# Contributing
+
+You are here to help on BPMN JS? Awesome, feel welcome and read the following guidelines in order to know how to contribute, to ask questions and to make BPMN JS such a great tool.
+
+All members of our community are expected to follow our [Code of Conduct](CODE_OF_CONDUCT.md). Please make sure you are welcoming and friendly in all of our spaces.
+
+## Contributions 
+
+There are many ways to contribute:
+
+- help people with the questions they ask on the [Github Issues](https://github.com/bonitasoft-labs/bpmn-js/issues )
+- submitting bug reports and feature requests in the [Github Issues](https://github.com/bonitasoft-labs/bpmn-js/issues/new )
+- [writing code](CONTRIBUTING.md#code-and-documentation-changes) which can be incorporated into BPMN JS itself
+- [improving](CONTRIBUTING.md#code-and-documentation-changes) the documentation
+- improve the existing example applications to demonstrate features in BPMN JS
+
+## Code and documentation changes
+
+For all contributions, please respect the following guidelines:
+
+1. If you've noticed a bug or have a feature request, let us know in the [GitHub Issue tracker](https://github.com/bonitasoft-labs/bpmn-js/issues/new )! So we can confirm the bug or approve your feature, and provide feedback, before starting to code :slightly_smiling_face:
+
+2. Do the changes in your own [fork](CONTRIBUTING.md#fork--create-a-branch) of the code
+
+3. Do not commit changes to files that are irrelevant to your feature or bugfix (eg: `.gitignore`).
+
+4. Provide [tests](CONTRIBUTING.md#running-tests) and documentation whenever possible.
+
+5. Be sure you have followed the [code style](CONTRIBUTING.md#code-style) for the project.
+
+6. Add a [changelog](CONTRIBUTING.md#add-a-changelog-entry) entry
```

Comment:
❓ I would prefer not manage the changelog with a file but through GitHub Releases